### PR TITLE
[RFR] automation for test_validate_lookup_button

### PR DESCRIPTION
**Pull request by john-dupuy**
{{ pytest: --use-provider vsphere65-nested cfme/tests/integration/test_ldap_auth.py }}

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 659a843a02b54d1835e8472c6a9757ccb0b6bf8a

 The *global* fixture **`ldap_auth_provider`** was changed, but I didn't find where it's used.
The *global* fixture **`setup_ldap_auth_provider`** is used in the following files:
- **cfme/tests/integration/test_ldap_auth.py**
    - *test_validate_lookup_button_provisioning*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit b3b1245a8bd7615c083b9dafe18b5be2545ae15f

 The *global* fixture **`ldap_auth_provider`** was changed, but I didn't find where it's used.
 The *global* fixture **`setup_ldap_auth_provider`** was changed, but I didn't find where it's used.
Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/integration/test_ldap_auth.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        "provider_name": provider.name,
+    })
+    # now enter email and lookup the first and last name
+    view.form.request.fill({
+        "email": f"{username}@{domain}"
+    })
+    # lookup the user's info
+    assert not view.form.request.lookup.disabled
+    view.form.request.lookup.click()
+    # to see info you have to change tabs
```
Comment (mshriver) [inline cfme/tests/integration/test_ldap_auth.py]: TODO: UX BZ?

Comment (john-dupuy) [reply cfme/tests/integration/test_ldap_auth.py]: Yup, I'll write that this afternoon

**Pull request merged**